.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+# TODO: Support running all integration tests when it's possible to 
+# run self-hosted runners in parallel.
+name: Integration Test
+
+on:
+  pull_request:
+
+jobs:
+  changed_packages:
+    runs-on: [self-hosted, linux]
+    if: ${{ github.repository_owner == 'flutter-tizen' && github.event_name == 'pull_request' }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Install flutter-tizen
+        run: |
+          git clone --depth=1 https://github.com/flutter-tizen/flutter-tizen.git
+      - name: Run integration tests
+        run: |
+          export PATH=`pwd`/flutter-tizen/bin:$PATH
+          `pwd`/tools/run_command.py test \
```

Comment:
```suggestion
          ./tools/run_command.py test \
```

Reply:
Ditto.

https://github.com/flutter-tizen/plugins/blob/master/.github/workflows/build.yml#L74

@bbrto21

Reply:
As far as I remember the `pwd` in the for loop has a reason. Without `pwd`, the loop will fail due to the presence of `cd $d`.

Reply:
Ah yes, realized it a few minutes ago and deleted the link.